### DISCUSSION TITLE
Appreciation and Inquiry about Mathesar's Next Release

### DISCUSSION THREAD

**Discussion by MrChadMWood** [mathesar-foundation/mathesar#3374] [Miscellaneous]
Hello Mathesar Team,

I'd like to express my gratitude for the incredible work you've done on Mathesar. It has proven to be an invaluable tool, addressing a unique use case for me that other software couldn't quite fulfill. One standout feature is its seamless integration with existing databases, without imposing alterations to the schema. As a data engineer, I appreciate the ability to retain control over my models and make intentional changes rather than having them automatically adjusted for the sake of application compatibility. 

Mathesar's ability to abstract the complexities of database usage while preserving the original schema aligns perfectly with my vision of an ideal database frontend. I do note the addition of the `mathesar_types` schema, but this is a minor deviation compared to the alternatives I've dealt with while using competing software.

For me its the combination of abstraction, such as being able to display alternative columns from parent tables instead of foreign key values, as well as retaining the original database model, that makes Mathesar so great. As an admin, I plan to lock down the schema for my Mathesar users such that they can only add or remove data from existing tables.

I've followed the discussions and updates on the next release of Mathesar.  As I await the upcoming release, I've held off on introducing Mathesar to my non-technical users, anticipating that the new features may enhance its maturity. I'd like to introduce my users to Mathesar once using it is a little more friction-less. I'm optimistic that this update will serve as a breakthrough, and I'm excited about the strides Mathesar is making.

For the sake of my own curiosity, could you provide insights into the current plans for this next release? Are we anticipating the next update's rollout in the coming weeks/months, or is a more extended timeline expected? I understand the significance of the thorough work being done, and I appreciate your dedication to delivering quality software. By no means do I want you to feel I am being impatient.

Thanks for any updates you can provide.

**Comment by kgodey** [comment]
Thank you very much for all the kind words, @MrChadMWood :). You're correct that we're focusing our upcoming releases on making Mathesar more friction-less and mature, and get us from alpha to beta.

We expect our next release (0.1.4) to be out sometime in January. The work is almost complete – we just need to finish up review and testing. 0.1.4 has taken us a lot longer than originally anticipated because we're making significant changes to the underlying infrastructure of our installation process. We had to do it all at once and test a lot of edge cases (since installation is something we _really_ don't want to break). 

Once this release is done, we'll be in much better shape to do more frequent releases. We're aiming to do a new release every month. It may take us a little bit of iteration to get there, though.

I hope this is helpful, please feel free to ask follow up questions! We're on holiday break until Jan 2, so we probably won't respond until then.

**Comment by kgodey** [comment]
Quick followup to say that we released v0.1.4 in January and v0.1.5 today. Our plan is to continue doing a new release at the end of each month.
